### PR TITLE
docs: add contributing

### PR DESCRIPTION
**Pull request by lumynou5**
# Description

Add contributing.

https://github.com/Lipoic/Lipoic-Backend/issues/39

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+| `style`    | Code style              |
+| `docs`     | Documentation           |
+| `ci`       | Continuous integration  |
+| `chore`    | Chore                   |
+| `revert`   | Commit reverting        |
+
+A exclamation mark (`!`) is added for breaking changes.
+
+### Code style
+Basically, use the prettier.
```
Comment (SiongSng) [inline CONTRIBUTING.md]: 怕有的人不知到什麼是 prettier，我建議提供更完整的說明。

```suggestion
Basically, use [Prettier](https://prettier.io/) to reformat the code.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Contributing
+First, thank you for your interest in contributing to Lipoic!
+We appreciate all kinds of contributions.
+If you have any question, please feel free and ask on [Discord](https://discord.gg/ArKk54ajfr) or
```
Comment (SiongSng) [inline CONTRIBUTING.md]:
```suggestion
If you have any questions, please feel free and ask on [Discord](https://discord.gg/ArKk54ajfr) or
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- Fork the repository.
+- Make changes and follow our [guidelines](#contributing-guidelines).
+- Issue a pull request.
+
+Please note that unfinished pull requests should be marked as drafts.
+
+## Contributing guidelines
+
+### Commits
+A commit should focus on one thing only. If necessary, update associated documents and tests in the same commit.
```
Comment (SiongSng) [inline CONTRIBUTING.md]: 感覺用 `each` 會更好？

```suggestion
Each commit should focus on one thing only. If necessary, update associated documents and tests in the same commit.
```

**Review by SiongSng — CHANGES_REQUESTED** [review]
非常棒的貢獻指南！

我覺得也可以在 README 裡面放入貢獻的章節並指向這個檔案

File: CONTRIBUTING.md (hunk 3 above)
Comment (lumynou5) [reply CONTRIBUTING.md]: yep

File: CONTRIBUTING.md (hunk 1 above)
Comment (lumynou5) [reply CONTRIBUTING.md]: How about "use Prettier to reformat the code"? If Prettier is it's name.

Comment (SiongSng) [reply CONTRIBUTING.md]: 好阿～

**Review by SiongSng — APPROVED** [review]
LGTM

**Pull request merged**